internal/release/promote/github_codeowners_test.go

```diff
@@ -0,0 +1,35 @@
+package promote
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPromotion(t *testing.T) {
+	for _, tc := range []struct {
+		name     string
+		content  string
+		expected []string
+	}{
+		{
+			name:     "simple",
+			content:  "src/ @damethot @synhaptein @pecusson @mdpaquin",
```

Comment:
Don't think it's a big issue to expose those via a public repo, but maybe sanitize user names with fake ones instead?